landoui/errorhandlers.py

```diff
@@ -94,7 +94,7 @@ def unexpected_error(e):
             title="Oops! Something went wrong.",
             message=(
                 "Something just went wrong. Try again or tell the team at "
-                "#lando on irc."
+                "#lando on Matrix."
```

Comment:
```suggestion
                "#conduit on Matrix (https://chat.mozilla.org/#/room/#conduit:mozilla.org)"
```

Reply:
#lando is an alias and we'd like users to be directed to the main #conduit room :)